nixos/modules/virtualisation/hyperv-guest.nix

```diff
@@ -22,6 +22,18 @@ in {
           Server 2008 R2 or earlier.
         '';
       };
+
+      enchanchedSessionMode = mkOption {
```

Comment:
mkEnableOption

Reply:
Cannot use `mkEnableOption`, since the option is disabled by default.

Reply:
`mkEnableOption` defaults to false too: 

```
  mkEnableOption =
    # Name for the created option
    name: mkOption {
    default = false;
    example = true;
    description = "Whether to enable ${name}.";
    type = lib.types.bool;
  };
```